cfme/tests/services/test_provision_stack.py

```diff
@@ -163,6 +164,8 @@ def test_provision_stack(appliance, stack, service_catalogs, request):
     stack.wait_for_exists()
 
 
+@pytest.mark.uncollectif(lambda: BZ(1575935, forced_streams=['5.8', '5.9']).blocks,
```

Comment:
Required: Please use @pytest.mark.meta(blockers=[BZ(1575935, forced_streams=['5.8', '5.9'])]) as this will be marked in run as blocked be bz